src/autoapp/Service/AndroidAutoEntity.cpp

```diff
@@ -238,6 +238,20 @@ void AndroidAutoEntity::onShutdownRequest(const aasdk::proto::messages::Shutdown
     controlServiceChannel_->sendShutdownResponse(response, std::move(promise));
 }
 
+void AndroidAutoEntity::onPingRequest(const aasdk::proto::messages::PingRequest& request)
+{
+    OPENAUTO_LOG(info) << "[AndroidAutoEntity] ping request ";
+    
+    auto promise = aasdk::channel::SendPromise::defer(strand_);
+    promise->then([]() {}, std::bind(&AndroidAutoEntity::onChannelError, this->shared_from_this(), std::placeholders::_1));
+    controlServiceChannel_->sendPingRequest(request, std::move(promise));
+}
+
+void AndroidAutoEntity::onVoiceSessionRequest(const aasdk::proto::messages::VoiceSessionRequest& request)
+{
+	OPENAUTO_LOG(error) << "[AndroidAutoEntity] voice session request not implemented";
```

Comment:
Note - I couldn't find anything that references "voice session" only video session and has more then just start and stop